scripts/robots/human/configuration.py

```diff
@@ -2,7 +2,9 @@
 
 # Model and ControlBoard
 robot_name = "Human"
-urdf_path = str(resolve_robotics_uri_py.resolve_robotics_uri("package://HDERviz/urdfs/humanSubject03_48dof.urdf"))
+# urdf_path = str(resolve_robotics_uri_py.resolve_robotics_uri("package://HDERviz/urdfs/humanSubject03_48dof.urdf"))
```

Comment:
Only minor comment I have. Feel free to ignore. Should we restore the use of resolve-uri instead of the hard-coded path ?